src/components/Network/NodeInfo/NodeInfoBox.jsx

```diff
@@ -13,6 +14,15 @@ import {
 } from '@fortawesome/free-solid-svg-icons'
 import i18n from '../../../i18n'
 
+library.add(
+  faShareAlt,
+  faUsers,
+  faBolt,
+  faLayerGroup,
+  faCloudDownloadAlt,
+  faClock
+)
```

Comment:
I removed `library` usage in a previous [PR](https://github.com/ethereum/ethereum-react-components/pull/76/files#diff-f020bbc7021b014262452e8ddfc740e1L5) for this reason:
>I believe the library function in fontawesome is best used when all icons are imported in some top-level component. Since we have a modular component lib, I think it makes sense to skip this convenience function.

Reply:
great thanks